openreview/venue/process/submission_revision_process.py

```diff
@@ -0,0 +1,3 @@
+def process(client, edit, invitation):
+
```

Comment:
what should happen here? I guess update the author group if the authorids change.

Reply:
yes, and I am sending an email to authors